### DISCUSSION TITLE
Add  Project-Kaleidoscope [skip ci] #11922

### DISCUSSION THREAD

**Discussion by ghost** [ROM-builders/temporary#11926] [Q&A]
Sir please merge this Project-Kaleidoscope [skip ci] #11922   
@Apon77 can you please add this sir

**Comment by Apon77** [comment]
What's the base of this rom? and what's the specialty in this rom?

**Reply by ghost** [reply]
Sir Minimal Android 12 rom based on aosp very minimalist and light rom with dot os Style Qs panel
Sir you check more about this project if you want to Github-https://github.com/Project-Kaleidoscope/android_manifest
telegram-https://t.me/KaleidoscopeOS

**Reply by Apon77** [reply]
ok resolve conflict and do pr

**Reply by ghost** [reply]
Did sir  #12059
